doc/user/how-to/metadata.md

```diff
@@ -21,32 +25,20 @@ that are in your QGIS project.
 
 Right click on the layer and select the **Properties** option. This will open the Layer Properties window.
 Next, find the metadata tab on the left side window panel. This will show any available information
-that is associate with the data layer, including but not limited to title and abstract.
+that is associated with the data layer, including but not limited to title and abstract.
 
 ![metadata.png](/_images/metadata.png)
 
-## Using the QGreenland Custom plugin
-
-```{attention}
-QGreenland Custom is no longer supported.
-```
+## Via Text Files in Layer Directories
 
-If you have the QGreenland Custom QGIS plugin, you can access QGreenland layer metadata through the plugin.
-First you will need download the layer of interest if you do not already have it downloaded. If you need help installing or downloading data using QGreenland Custom, please follow 
-[this how-to guide](https://qgreenland-plugin.readthedocs.io/en/latest/user-how-to/plugin-how-to.html) first. 
+In each layer directory inside the QGreenland package, you should see 2 text
+files, one **provenance.txt**, which includes information on how the data is
+processed, and one **metadata.txt**.
 
-Access the plugin via the **Web** tab in the top **Menu Bar**, and select the **Manage Data** option.
-This will open a window showing any layers you have downloaded via the plugin. 
+Open up **metadata.txt** to read through the layer metadata including layer
+description, dataset abstract, and citation information.
 
-![add_to_project_plugin.jpg](/_images/add_to_project_plugin.jpg)
-
-Click the **Explore files** button at the bottom left. This will open up a window to the download location
-that you selected when downloading data via the plugin. If you have only downloaded 1 layer via the plugin,
-you will only see 1 file directory here, otherwise, scroll through to find your layer of interest and open
-up that directory. In the layer directory, you should see 2 text files, one **provenance.txt**, which includes
-information on how the data is processed, and one **metadata.txt**. Open up **metadata.txt** to read through the layer metadata.
+Open the **provenance.txt** file to see all of the processing steps applied to
+the data for QGreenland.
 
 ![layer_dir.png](/_images/layer_dir.png)
-
-You have just learned 2 ways in which you can locate and access QGreenland layer metadata. For more
-information on layer properties and metadata, read our **Interacting with Geospatial Data in QGreenland Core** [documentation](https://qgreenland.readthedocs.io/en/latest/tutorials/interacting-with-geospatial-data.html).
```

Comment:
Thinking out loud some more: It'd be nice for our build process to warn when this type of link is detected. We should not have any references to our deployment URL in our docs.

Reply:
:100: !